t/20-in-pg.t

```diff
@@ -0,0 +1,48 @@
+use Red;
+use Test;
+
+my $*RED-PG-TEST-DB = $_ with %*ENV<RED_PG_TEST_DB>;
+
+my $*RED-DEBUG          = $_ with %*ENV<RED_DEBUG>;
+my $*RED-DEBUG-AST      = $_ with %*ENV<RED_DEBUG_AST>;
+
+plan 2;
+unless $*RED-PG-TEST-DB {
+    "No RED_PG_TEST_DB initialized.".say;
+    skip-rest 2;
+}
+
+model Category is table<test_category> {
+    has Int $.id is serial;
+    has Int $.parent_id is column{ :references{ Category.id }, :nullable, };
```

Comment:
This test is failing here. For `Red:api<2>` you need to set the `:model<>` for `is referencing` or `:model-name<>` in this case... and receive the model type as parameter:
```perl6
has Int $.parent_if is column{ :references{ .id }, :nullable };
```